modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_plain_text.yml

```diff
@@ -0,0 +1,33 @@
+id: az_paragraph_plain_text
+label: AZ Pragrapgh Plain Text
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: d7_paragraphs_item
+  bundle: uaqs_plain_text
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_text
+
+process:
+
+  field_az_text_area:
+    plugin: sub_process
+    source: field_uaqs_text_area
+    revision_id: revision_id
+    process:
+      delta: delta
+      value: value
+      summary: summary
```

Comment:
`field_az_text_area` in the `az_text` paragraph does not have the summary subfield enabled so this probably should be omitted.